en/tutorials/linux_sbus.md

```diff
@@ -0,0 +1,35 @@
+# Sbus Driver for linux（drivers/linux_sbus）
+This driver allows the autopilot to fetch data from the futaba sbus receiver via serial port, up to 16 channels.  
+Onboard serial, usbtty, and other types of serial ports are supported.  
+All platforms running linux system are supported.  
+
+## Required components
+
+NPN transistor * 1  
+10K resistance * 1  
+1K resistor * 1  
+
+## Optional components
+Usbtty *1  
+
+## Installation
+Connect the components according to the following ways  
+
+S.bus singal->1K resistor->NPN transistor base  
+NPN transistor emmit -> GND  
+3.3v  VCC-> 10K resistor -> NPN transistor collection ->usbtty rxd  
+5.0v  VCC->S.bus vcc  
+GND ->S.bus gnd  
+
```

Comment:
I did not make a PCB for this, just a handmade circuit.  I will draw a hand-painted circuit.
In fact this driver is also for "posx_rpi_no_shield"，It‘s inconvenient to buy pixfmini or navio in China. Since Last time I submitted the relevant code, many Chinese users began to  use the rpi to build the aircraft without shield. 

http://bbs.5imx.com/forum.php?mod=viewthread&tid=1252361&extra=page%3D2